README.md

```diff
@@ -145,6 +145,16 @@ python3 app.py
 ```
 > **Note**: The server should indicate it is running on http://127.0.0.1:8080. Visit this address in your browser and submit a URL to begin.
 
+### Configuring the Number of Threads
+
```

Comment:
We can summarize this with a note at the end of prerequisites to say, that the user can adjust this via the `config.json` file.